.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
```

Comment:
I would have liked to include `os: [ubuntu-latest, macos-latest]` but the build fails on mac for a weird reason. See https://github.com/FrNecas/2ls/runs/3381425349#step:3:196 :
```
 In file included from xml_parser.cpp:13:
In file included from /Applications/Xcode_12.4.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/fstream:188:
In file included from /Applications/Xcode_12.4.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/filesystem:240:
/Applications/Xcode_12.4.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/stack:127:5: error: expected ')'
    stack()
    ^
../util/parser.h:144:25: note: expanded from macro 'stack'
#define stack(x) (PARSER.stack[x])
                        ^
/Applications/Xcode_12.4.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/stack:127:5: note: to match this '('
../util/parser.h:144:18: note: expanded from macro 'stack'
#define stack(x) (PARSER.stack[x])
                 ^
```
That looks like fstream include fails due to some problem in the C++ library on Mac? I can't reproduce, test or debug it since I don't have a mac :/ So I put only Ubuntu there for now.